pkg/operation/botanist/component/vpnseedserver/vpn_seed_server.go

```diff
@@ -0,0 +1,645 @@
+// Copyright (c) 2021 SAP SE or an SAP affiliate company. All rights reserved. This file is licensed under the Apache Software License, v. 2 except as noted otherwise in the LICENSE file
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package vpnseedserver
+
+import (
+	"context"
+	"fmt"
+
+	v1beta1constants "github.com/gardener/gardener/pkg/apis/core/v1beta1/constants"
+	"github.com/gardener/gardener/pkg/operation/botanist/component"
+	kutil "github.com/gardener/gardener/pkg/utils/kubernetes"
+	"github.com/gardener/gardener/pkg/utils/managedresources"
+	"github.com/gogo/protobuf/types"
```

Comment:
Put this import into the third group

Reply:
I am confused. Which import should go into which group?
The ones you quoted above are the second group. Do you want the second and third group to be merged?

Reply:
Usually, we have three import groups: 1) core package, 2) packages of the module, 3) 3rd party packages. Here, `"github.com/gogo/protobuf/types"` is part of the second group although it's a 3rd party package, so it should be moved down. :)

Reply:
Thanks a lot for the explanation. Now, it does make sense.
Adapted with the last commit.